Exceptions
----------

Problematic
^^^^^^^^^^^

How can we provide our very own exceptions ?

Documentation
^^^^^^^^^^^^^

.. automodule:: PyFunceble.exceptions
   :members:
   :private-members:

